coalahtml/_coalahtml/app/controllers/directives.js

```diff
@@ -16,12 +17,34 @@ angular.module('coalaHtmlApp')
           if (lineNumClass && lineNumClass[1] && lineNumClass[1].length > 0) {
             lineNums = +lineNumClass[1];  // Convert to integer
           }
-
           element.html(prettyPrintOne(newValue, '', lineNums));
+
+          // Idea is to create elements dynamically and append to the code block.
+          var _className = "highlight-line";
```

Comment:
Still, don't like this name. `_className` is too general. Maybe, `highlightLineClass` or something. Something that will tell what it means.